compact/definitions.xml

```diff
@@ -410,6 +410,50 @@ Examples:
     <constant name="SolenoidBarrel_rmin"       value="Solenoid_rmin"/>
     <constant name="SolenoidBarrel_length"     value="Solenoid_length"/>
 
+    <documentation>
+      ## IP6 Beam Pipe Parameters - Note might need cleaning
+    </documentation>
+    <constant name="CrossingAngle" value="-0.025*rad "/>
+    <constant name="ionCrossingAngle" value="CrossingAngle"/>
+    <constant name="electronCrossingAngle" value="0.0"/>
```

Comment:
Arguably, these are not *crossing* angles, since they refer to one beam only. Can we just call these "angle"?